app/src/layout/Combat.tsx

```diff
@@ -624,13 +640,23 @@ const Combat: React.FC<CombatProps> = (props) => {
                 </Button>
               </Link>
               {showNextMatch && arenaOpponentId && (
-                <Button
-                  id="return"
-                  className="basis-1/2 w-full"
-                  onClick={() => startArenaBattle({ aiId: arenaOpponentId })}
-                >
-                  Go Again
-                </Button>
+                <div>
+                  <Button
+                    id="return"
+                    className="basis-1/2 w-full"
+                    onClick={() => startArenaBattle({ aiId: arenaOpponentId })}
+                  >
+                    Go Again
+                  </Button>
+
+                  <Button
+                    id="heal-return"
+                    className="basis-1/2 w-full mt-1"
```

Comment:
An option for this which would avoid adding new endpoints could be to put in the ramen options on this page, e.g. generalize the menu components from `app/ramenshop/page.tsx` into the `layout` folder so that we could insert them directly on this page. Let's discuss with Neon

```typescript
<div className="grid grid-cols-3 text-center font-bold italic p-3">
          <MenuEntry
            title="Small Bowl"
            entry="small"
            image="/ramen/small_bowl.webp"
            userData={userData}
            onPurchase={() => mutate({ ramen: "small" })}
          />
          <MenuEntry
            title="Medium Bowl"
            entry="medium"
            image="/ramen/medium_bowl.webp"
            userData={userData}
            onPurchase={() => mutate({ ramen: "medium" })}
          />
          <MenuEntry
            title="Large Bowl"
            entry="large"
            image="/ramen/large_bowl.webp"
            userData={userData}
            onPurchase={() => mutate({ ramen: "large" })}
          />
```

Reply:
Neon wanted to keep the 500 ryo, although I will utilize this for battle pyramid.